sktime/regression/deep_learning/tapnet.py

```diff
@@ -207,8 +207,7 @@ def _fit(self, X, y):
         -------
         self: object
         """
-        if self.callbacks is None:
-            self._callbacks = []
+        self.callbacks = self.callbacks or []
```

Comment:
Same issue as in `CNNRegressor`

Reply:
Okay. I made the same changes to the deep regressors.